.github/workflows/test_derper.yml

```diff
@@ -0,0 +1,70 @@
+name: Deploy Test Derper
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 4 * * *'
+
+concurrency:
+    group: ${{ github.workflow }}-${{ github.ref }}
+    cancel-in-progress: true
+    
+env:
+    RUST_BACKTRACE: 1
+    RUSTFLAGS: -Dwarnings
+    RUSTDOCFLAGS: -Dwarnings
+    MSRV: "1.72"
+    SCCACHE_CACHE_SIZE: "50G"
+    
+jobs:
+    build_derper:
+        runs-on: [self-hosted, linux, X64]
+        if: github.ref_name=='main'
+        env:
+            # Using self-hosted runners so use local cache for sccache and
+            # not SCCACHE_GHA_ENABLED.
+            RUSTC_WRAPPER: "sccache"
+            RUST_BACKTRACE: full
+            RUSTV: stable
+        steps:
+        - uses: actions/checkout@v2
+        - name: Install rust
+          run: rustup toolchain install stable
+        - name: Install sccache
+          uses: mozilla-actions/sccache-action@v0.0.3
+    
+        - name: build release
+          run: |
+            cargo build --release --all-features --bin derper
+    
+        - name: Setup awscli on linux
+          run: |
+            curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
+            unzip awscliv2.zip
+            sudo ./aws/install --update
+        - name: Set aws credentials
+          run: |
+                echo "AWS_ACCESS_KEY_ID=${{secrets.S3_ACCESS_KEY_ID}}" >> $GITHUB_ENV
+                echo "AWS_SECRET_ACCESS_KEY=${{secrets.S3_ACCESS_KEY}}" >> $GITHUB_ENV
+                echo "AWS_DEFAULT_REGION=us-west-2" >> $GITHUB_ENV
+      
+        - name: push release
+          run: |
+              aws s3 cp ./target/release/derper s3://vorc/derper-linux-amd64-${GITHUB_SHA::7} --no-progress
+        
+
+    deploy_derper:
+        runs-on: ubuntu-latest
+        if: github.ref_name=='main'
+        needs: build_derper
+        steps:
+        - uses: actions/checkout@v2
+        - name: Run Staging Deploy Playbook
+          uses: arqu/action-ansible-playbook@master
```

Comment:
why forked?

Reply:
I don't trust random actions on other peoples repos and tied to main/master.

Reply:
:+1: though, should probably clone them to n0-computer org